### ISSUE TITLE
Please change ```#!/usr/local/bin/python``` to ```#!/usr/bin/env python```

### ISSUE THREAD

**Issue by shenwei356** [mbreese/qtask#1]
Please change python interpreter declaration `#!/usr/local/bin/python` to `#!/usr/bin/env python` in scripts. Because people may have installed a local python at their home.

**Comment by mbreese** [comment]
Where is this? 

All of the executable scripts in this repo that I see use `#!/usr/bin/env python`...

**Comment by shenwei356** [comment]
Sorry, I'm confused. It's so weird.

I met some trouble in using qtask. It's hard to configure for PBS schedulers, maybe you can write example for that. 

For a `.qtaskrc` file with content:

```
runner=pbs
```

Error ocured:

```
$ mqsub pwd
Traceback (most recent call last):
  File "/db/home/shenwei/.local/bin/mqsub", line 12, in <module>
    import qtask
  File "/db/home/shenwei/.local/lib/python2.7/site-packages/qtask/__init__.py", line 433, in <module>
    pipeline = __Pipeline()
  File "/db/home/shenwei/.local/lib/python2.7/site-packages/qtask/__init__.py", line 298, in __init__
    self.runner = PBS(**runnerconf)
  File "/db/home/shenwei/.local/lib/python2.7/site-packages/qtask/pbs.py", line 8, in __init__
    qtask.JobRunner.__init__(self, *args, **kwargs)
TypeError: __init__() got an unexpected keyword argument 'tmpdir'
```

I tried to edit `qtask.JobRunner.__init__`, but more error came out. So I gave up, and  wrote a little script [easy_qsub](https://github.com/shenwei356/easy_qsub) to submit jobs.

**Comment by mbreese** [comment]
You’re absolutely right… I haven’t tested this with PBS schedulers thoroughly at all. I no longer have access to a PBS-based cluster, so that’s the main limiting factor. I took older code that I had used for a PBS cluster and tried to migrate it. I deal mainly with SGE and SLURM clusters now...

Unless you’re working on programmatically defining complex pipelines, your easy_qsub is probably what you’re looking for anyway. Although, the “mqsub” script that I have is very helpful for submitting jobs for multiple input files based on a command-line template.
